src/app/client/src/app/modules/shared/services/config/url.config.json

```diff
@@ -204,12 +204,13 @@
     "BATCH": {
       "CREATE": "course/v1/batch/create",
       "UPDATE": "course/v1/batch/update",
-      "ADD_USERS":"course/v1/admin/enroll", 
+      "ADD_USERS": "course/v1/batch/user/add",
```

Comment:
@sanjeev-scm @paritshivani can u guys please confirm this change @snehalsabade

Reply:
I have updated the paths. Please review and merge

Reply:
I have updated the API paths. Please review and merge